docs/index.rst

```diff
@@ -52,6 +54,43 @@ Why MembraneCurvature?
 in Python 3 to derive 2D maps of membrane curvature from MD Simulations, using the most recent version of `MDAnalysis`_ 
 Are you interested in calculating mean and Gaussian curvature from MD simulations? This tool is for you!
 
+
+Installation
+--------------
+
+The main dependency in MembraneCurvature is `MDAnalysis`_. You can find
+instructions to install the latest stable version of MDAnalysis via `conda`_ in the `User Guide`_.
+
+MembraneCurvature is available via pip:
+
+.. code-block:: bash
+
+   pip install membrane-curvature
+
+To install from source:
+
+.. code-block:: sh
```

Comment:
why not also bash?

Reply:
I think I added this very last-minute change to check how does it look vs bash and forgot to revert changes. I'll leave it as bash, thanks!